oops/gold_master/__init__.py

```
@@ -1147,31 +1147,19 @@ def run_tests(self):
-            localpath = None
-            try:
-                localpath = log_path.retrieve()
-            except FileNotFoundError:
-                pass
-
-            if localpath:
+            localpath = log_path.get_local_path()
+            if os.path.exists(localpath):
+                
                 # Append the latest modification date to the pre-existing file
                 dt = datetime.datetime.fromtimestamp(os.path.getmtime(localpath))
```

Ruff (0.15.12)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)